libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -220,6 +220,7 @@ method rpcHandler(g: GossipSub,
             if msgs.len > 0:
               trace "forwarding message to", peerId = id
               sent.add(g.peers[p].send(@[RPCMsg(messages: msgs)]))
+            # TODO FIXME what to do with erros here??
```

Comment:
@dryajov Should we just leave `allFutures` and let it throw here? (repeating the question few times around)

Reply:
same as previous